Debian package missing flake8 dependency

Related to https://github.com/mvo5/squid-deb-proxy/commit/da49a8bc28189993e08d04205d027d31e2480387

```
flake8 apt-avahi-discover
make[1]: flake8: No such file or directory
make[1]: *** [Makefile:7: check] Error 127
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
dh_auto_test: error: make -j16 check returned exit code 2
make: *** [debian/rules:6: build] Error 25
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
```

--- a/debian/control
+++ b/debian/control
@@ -6,7 +6,8 @@ Build-Depends: debhelper (>= 12),
                gettext,
                intltool,
                pkg-config,
-               po-debconf
+               po-debconf,
+               flake8
 Standards-Version: 4.4.0
 Vcs-Bzr: https://code.launchpad.net/~squid-deb-proxy-developers/squid-deb-proxy/trunk
 Homepage: https://launchpad.net/squid-deb-proxy

